### ISSUE TITLE
[Bug] [a11y]: Keyboard focus not visible for table in Windows High contrast mode settings

### ISSUE THREAD

**Issue by vdevarapally** [microsoft/fluentui#27579]
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
"fluentui/react-northstar": "^0.65.0",
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://fluentsite.z22.web.core.windows.net/0.66.2/maximize/table-example-navigable-shorthand/false

### Bug Description

Prerequisites - Windows Settings -> Accessibility -> Contrast Themes -> Select Aquatic High Contrast Theme

Using tab key , trying to focus on the table. But the focus is not visible.
## Actual Behavior

Keyboard focus is not visible to the user when the focus is on the table row/column with no borders.

## Expected Behavior
Keyboard focus should be visible to the user even with Windows High contrast theme.

### Logs

```shell
No logs
```

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
It looks like this is caused by having transparent borders on all the cells. The focus should be visible if the rest state borders are removed -- IMO they're not any more necessary in high contrast mode than they are outside of it.

**Comment by vdevarapally** [comment]
@smhigley Got it. Shouldn't this be handled by Fluent UI ?

**Comment by smhigley** [comment]
@vdevarapally since this is a Fluent Northstar component, I'll let that team comment on whether there will be a change to the component itself. Looks like @jurokapsiar was just assigned, he'd know whether there's any background to the table border styles.

**Comment by kippllo** [comment]
This is also an issue with drop-down when in a Windows high-contrast theme.

Without focus:
<img width="211" alt="Screenshot 2023-05-08 123902" src="https://user-images.githubusercontent.com/23347678/236894919-43ddedac-9fb6-491d-8621-945e7449c18b.png">

With focus:
<img width="213" alt="Screenshot 2023-05-08 123915" src="https://user-images.githubusercontent.com/23347678/236895038-f04984b7-3fc4-4d88-b28c-3aa0a16a54f3.png">

**Comment by kippllo** [comment]
Also, buttons inside of table cells have bad focus too.
